packages/docs/webpack.config.js

```diff
@@ -10,7 +10,7 @@ const webpack = require('webpack');
  * @param {Boolean} hotReload - Enable Webpack's hot module replacement
  * @return {Object} Webpack config
  */
-function createConfig(docsPath, rootPath = '', srcPaths, hotReload = true) {
+function createConfig(docsPath, srcPaths, githubUrl, rootPath = '', hotReload = true) {
```

Comment:
This function will likely need further changes, and I've also been debating with myself whether this should live here in the docs package or in the scripts package.

Reply:
what are some of the pros of keeping it in the docs package?

Reply:
Answered this in a comment above except that it wasn't pros so much as a weak sippery slope argument against it. I think it's totally reasonable to move it into scripts.